### PR TITLE
Fixed issues with wrong text for currency in Polish language

### PR DESCRIPTION
**Pull request by kszys**
When the amount to be converted to text using to_currency function in Polish language was ending with a number of decimals being a whole 10 - e.g., 11.30, 123.40, 34234.80, etc. the resulting text was saying respectively _three_, _four_, _eight_ (in Polish) rather than _thirty_, _forty_, _eighty_. This was because the same conversion was used for regular numbers as for _cents_, which is incorrect. This patch fixes it for Polish language.

### CODE REVIEW

**Comment by erozqba** [comment]
Hi @kszys 
Thanks a lot for your contribution, could you also include a test in you PR to avoid future regressions? Right now we don't have any test for Polish and will be great to start testing it.

**Comment by kszys** [comment]
I added the unittests, but the checks seem to fail... The tests worked fine, when I tested locally - not sure what I need to do now...

**Comment by erozqba** [comment]
Hi @kszys 
Thanks a lot for the unit tests, looking at it, the problems is that there is not python 3 compatible. All tests pass for python 2.7, but we are running tests in 4 different python versions 2.7, 3.4, 3.5, 3.6. 

For more information, you can look at https://travis-ci.org/savoirfairelinux/num2words/jobs/205850020
The fails is in 

> test_currency

Thanks in advance if you can look at it and fix it.
I will try to look at it later when I find some time.

**Comment by kszys** [comment]
Ok. It seems I have not tested with python 3 :)
I will take a look later today.

**Comment by kszys** [comment]
This should do it :)

File: num2words/lang_PL.py [hunk 1]
```diff
@@ -239,12 +245,14 @@ def to_currency(n, currency='EUR', cents=True, seperator=','):
...
         n = abs(n)
-        left = n / 100
+        left = n // 100
         right = n % 100
     else:
         n = str(n).replace(',', '.')
         if '.' in n:
             left, right = n.split('.')
+            if len(right)==1:
+                right = right+'0'
```
Comment (erozqba) [inline num2words/lang_PL.py]: Could you add whitespace around the operators? So we keep respecting PEP8?
`if len(right) == 1:` and `right = right + '0'`

**Comment by erozqba** [comment]
Hi @kszys thanks for your work! We will change tox in the future so it will automatically check for PEP8

**Comment by kszys** [comment]
So when is this going to be pushed in a new version of the module that I can install with `pip`? I need it working on a few systems ;)

**Comment by erozqba** [comment]
Yes, @hsoft is usually in charge of the releases, he will probably make a new one when he finds some free time. :+1:

**Pull request merged**